From 6cff4b34ab52b8178e2f7b1268a7ef35f2fb908c Mon Sep 17 00:00:00 2001
From: Dexter Gaon-Shatford <dexter@gaonshatford.ca>
Date: Sat, 20 Jul 2024 23:31:32 -0400
Subject: [PATCH] Include limits.h in Process.cpp

Without this header, the PATH_MAX macro isn't defined when building
against musl libc.
---
 src/Utils/Process.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/Utils/Process.cpp b/src/Utils/Process.cpp
index 5688fb1..3f0d48c 100644
--- a/src/Utils/Process.cpp
+++ b/src/Utils/Process.cpp
@@ -27,0 +28 @@
+#include <limits.h>
-- 
2.45.2

